README.md

```diff
@@ -35,6 +45,50 @@ ddf = dask_bigquery.read_gbq(
 
 ddf.head()
 ```
+## Example: write to BigQuery
+
+With default credentials:
+
+```python
+import random
+import pandas as pd
+import dask.dataframe as dd
+import dask_bigquery
+from datetime import datetime, timedelta
+
+df = pd.DataFrame({
```

Comment:
in #53 I was just using `dask.datasets.timeseries()`, same idea but slightly more concise?

Reply:
You're fast @bnaul. :) 🚀 Good idea, I'll use that instead.

Reply:
Now using timeseries.